Integrations/QuinC/README.md

```diff
@@ -0,0 +1,314 @@
+            ], 
+            "Timestamp": "2019-10-04T22:20:21Z", 
+            "ServicesDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "JobId": "2e667db2-ee5a-4522-aff8-4047769bd211", 
+            "Volume": false, 
+            "Registry": false, 
+            "Services": false, 
+            "UsbDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "RoutingDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "ArpDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "CertificateDetails": {
+                "Code": 0, 
+                "File": "\\\\X.X.X.X\\D$\\Program Files\\AccessData\\QuinC\\app\\demo\\Demo Case\\c00a2abf-1076-412b-8dea-67305fb8015f\\Jobs\\job_854\\9645eb6b-5bc5-4342-823b-8aa0e4594c27\\1\\certificates.xml"
+            }, 
+            "PrefetchDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "VolumeDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "LiveRegistry": false, 
+            "ResultType": 18, 
+            "Snapshot": true, 
+            "Prefetch": false, 
+            "OperationType": 12, 
+            "DNSCache": false, 
+            "DriversDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "TasksDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "Arp": false, 
+            "Tasks": false, 
+            "UserMessage": null, 
+            "NIC": false, 
+            "RegistryDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "TaskId": "9645eb6b-5bc5-4342-823b-8aa0e4594c27", 
+            "LiveRegistryDetails": {
+                "Code": 0, 
+                "File": null
+            }, 
+            "SnapshotDetails": {
+                "Code": 0, 
+                "File": "\\\\X.X.X.X\\D$\\Program Files\\AccessData\\QuinC\\app\\demo\\Demo Case\\c00a2abf-1076-412b-8dea-67305fb8015f\\Jobs\\job_854\\9645eb6b-5bc5-4342-823b-8aa0e4594c27\\1\\snapshot.xml"
+            }, 
+            "Message": null, 
+            "UsersDetails": {
+                "Code": 0, 
+                "File": null
+            }
+        }
+    }
+}
+```
+
+##### Human Readable Output
+Job completed successfully
+
+### accessdata-legacyagent-get-processlist
+---
+Return list of process from legacy agent
+##### Base Command
+
+`accessdata-legacyagent-get-processlist`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| caseid | ID of case | Required | 
+| target_ip | IP address of agent | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Accessdata.Job(val.Type=="Volatile").ID | string | ID of job | 
```

Comment:
this should be just `Accessdata.Job.ID` and also include `Accessdata.Job.Type`
because your output json includes
```
{
    "Accessdata.Job": {
        "Type": "Volatile", 
        "ID": 857
    }
}